build-tools/automation/azure-pipelines-oss.yaml

```diff
@@ -2,66 +2,186 @@
 
 name: $(Build.SourceBranchName)-$(Build.SourceVersion)-$(Rev:r)
 
-trigger:
-  - master
-  - d16-*
+# Triggers are enabled on the pipeine definition in place of being defined here
+#  https://dev.azure.com/xamarin/public/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=48&view=Tab_Triggers
+# trigger:
+#  - master
+#  - d16-*
```

Comment:
Do we need to override these YAML defined triggers? Is this what allows us to get automatic triggers in both Azure DevOps organizations?

Reply:
Agree that incorporating the trigger settings within the YAML file itself provides greater flexibility and documention.  Addressed with commit f2783df6fb499a19a4da9742ec2981dd15bdd6e6